Tests/OctoKitTests/UserTests.swift

```diff
@@ -95,6 +95,30 @@ class UserTests: XCTestCase {
         XCTAssertEqual(subject.numberOfPublicRepos, 6)
         XCTAssertEqual(subject.numberOfPublicGists, 10)
         XCTAssertEqual(subject.numberOfPrivateRepos, 4)
+        XCTAssertNil(subject.nodeId)
+        XCTAssertEqual(subject.htmlURL, "https://github.com/pietbrauer")
+        XCTAssertEqual(subject.followersURL, "https://api.github.com/users/pietbrauer/followers")
+        XCTAssertEqual(subject.followingURL, "https://api.github.com/users/pietbrauer/following{/other_user}")
+        XCTAssertEqual(subject.gistsURL, "https://api.github.com/users/pietbrauer/gists{/gist_id}")
+        XCTAssertEqual(subject.starredURL, "https://api.github.com/users/pietbrauer/starred{/owner}{/repo}")
+        XCTAssertEqual(subject.subscriptionsURL, "https://api.github.com/users/pietbrauer/subscriptions")
+        XCTAssertEqual(subject.reposURL, "https://api.github.com/users/pietbrauer/repos")
+        XCTAssertEqual(subject.eventsURL, "https://api.github.com/users/pietbrauer/events{/privacy}")
+        XCTAssertEqual(subject.receivedEventsURL, "https://api.github.com/users/pietbrauer/received_events")
+        XCTAssertFalse(subject.siteAdmin!)
+        XCTAssertTrue(subject.hireable!)
+        XCTAssertNil(subject.bio)
```

Comment:
Currently I am not sure if this is mapped correctly or if the field is empty in the JSON. Could you change the stub and add a `bio`, `twitterUsername` and `noteId`?